admin/views/error.ejs

```diff
@@ -4,4 +4,8 @@
 
 <h2>Error code: <%= errorCode %></h2>
 <p>ID: <%= errorId %></p>
+<% if (typeof mtcInfo !== 'undefined') { %>
+<h2>MTC Information</h2>
```

Comment:
i dont think we want an 'MTC Information' header for this.  just a paragraph element for the info will be fine.

Reply:
I had made it that you add an `mtcInfo` property. Instead, I’ll make that an object, so you could do something like:
```
error.mtcInfo = {
  'HELPDESK': 'Error for helpdesk users to see',
  'TEACHER': 'Error for teacher users to see',
  ... other roles
}
```
It’ll then show the error based on the user’s role.

Reply:
that's a nice addition, thanks 👍

Reply:
@danwilliams could you have a 'default/catch all' that shows for any kind of user please